appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# AppVeyor file
+# http://www.appveyor.com/docs/appveyor-yml
+
+# Build version format
+version: "{build}"
+
+# Fix line endings on Windows
+init:
+  - git config --global core.autocrlf true
```

Comment:
I'm not sure we want this. `npm install` doesn't change line endings, it seems like we shouldn't enable this.